fused-effects.cabal

```diff
@@ -51,10 +54,10 @@ library
                      , Control.Effect.Sum
                      , Control.Effect.Trace
                      , Control.Effect.Writer
-  build-depends:       base >=4.9 && <4.13
-                     , deepseq >=1.4.3 && <1.5
-                     , MonadRandom >=0.5 && <0.6
-                     , unliftio-core >= 0.1.2
+  build-depends:       base           >= 4.9 && < 4.14
```

Comment:
We should probably bump minimum base to 4.10 (or whatever shipped with 8.4) so that Hackage doesn’t try to build configurations with old GHCs.